cmd/verify.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/redhat-certification/chart-verifier/pkg/chartverifier"
 )
 
+const (
+	Version = "1.0.0"
+)
```

Comment:
One last comment!
I think we can override the version number when building the program.  No need to change this file every time.  To support that, let's use `var` instead of `const` for `Version`. This way we can change the Version number during build time like this:
```
go build -ldflags="-X 'github.com/redhat-certification/chart-verifier/cmd.Version=1.2.0'"
```

Reply:
Done.